frontend/public/components/dashboards-page/overview-dashboard/top-consumers-card.tsx

```diff
@@ -56,20 +66,39 @@ const getTopConsumersQueries = (): TopConsumersMap => {
   return topConsumers;
 };
 
-const TopConsumersCard_: React.FC<DashboardItemProps> = ({
+const getResourceToWatch = (model: K8sKind) => ({
+  isList: true,
+  kind: model.crd ? referenceForModel(model) : model.kind,
+  prop: 'consumers',
+});
+
+const BarLabelComponent: React.FC<BarLabelComponentProps> = React.memo(({ model, title, namespace }) => (
```

Comment:
Why not simply `BarLabel` ?

(On the other hand, in terms of props, using e.g. `LabelComponent` is perfectly fine IMO.)

Reply:
right,`Component` suffix is unnecessary, renamed